scripts/publish-root.gradle

```diff
@@ -0,0 +1,34 @@
+// Create variables with empty default values
```

Comment:
I don't know if `publish-root` has some meaning to it but for the person who doesn't know it, it sounds very cryptic.
Maybe some more clear name could be used. For example: `publish-settings`, `publish-config`, `publish-properties`, `publish-variable`...

Reply:
Renamed to `publish-settings`